src/Menu.java

```diff
@@ -0,0 +1,140 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+public class Menu {
+
+
+    public static void main(String[] args){
+
+        try {
+            int userInput;
+            do {
+                Utils.clearScreen();
+                BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+                Utils.println("starting_menu",1,2,3,4, Game.isSoundOn()? "Off": "On",5,6);
```

Comment:
This works for what we need now, but f you would like to improve it then you can add the the options to a json file and create a `Settings.getJSONArray()` method. Then you can iterate the array like normal without hard coding the numbers. It would look something like this:
```suggestion
JSONArray menu = Settings.getArray("starting_menu");
for(int i = 0; i < menu.length(); i++){
    Utils.println("display_shop_item", i+1, menu.get(i));
}
```